extensions/amp-sona/validator-amp-sona.protoascii

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```

Comment:
Can you update this value to 2020? The value should match the year the new file is committed.

Reply:
👍